src/gc.h

```diff
@@ -519,18 +520,18 @@ void gc_final_pause_end(int64_t t0, int64_t tend);
 #endif
 
 #ifdef GC_TIME
-void gc_time_pool_start(void);
-void gc_time_count_page(int freedall, int pg_skpd);
-void gc_time_pool_end(int sweep_full);
-void gc_time_sysimg_end(uint64_t t0);
+void gc_time_pool_start(void) JL_NOTSAFEPOINT;
```

Comment:
Does your code actually look into the gc internal functions?

Reply:
yes, it looks at everything.